www/css/main.diary.css

```diff
@@ -494,19 +494,22 @@ enketo-notes-list, .notes-list {
 }
 
 .notes-list {
-  padding: 0 5px 5px 5px;
+  padding: 0 15px 5px 15px;
```

Comment:
To me, this puts the trash can a little too far in. How about 15 on the left and 10 on the right?
```suggestion
  padding: 0 10px 5px 15px;
```
Here's what that looks like
<img width="340" alt="image" src="https://user-images.githubusercontent.com/15843932/226075714-d48d867a-61a7-40a6-84ed-3534f41872c9.png">